pywbem/cim_operations.py

```diff
@@ -453,11 +459,23 @@ class WBEMConnection(object):
         :class:`~pywbem.BaseOperationRecorder`, each operation
         that is executed on this connection will be recorded by
         invoking its :meth:`~pywbem.BaseOperationRecorder.record` method.
+
+      use_pull_operations (:class:`py:bool`):
+        Attribute that defines whether the WBEM server defined by this
+        connection has or is to use the pull operations
+
+        if `None` no pull operation has been attempted on this connection
+        If `True` the user has specified pull operations only or the client
+        has received a successful response to a Iter... method.
+        If `False` the  the user set use_pull_operations=False in the WBEM
+        connection creation or the server has responded with not supported
+        indicating that the pull operations are not supported on the server.
+
```

Comment:
It was originally just internal, there has ... When we extended it to cover both what the client sees and what the user wants, we sort of mixed things up.

Done